### PR TITLE
Add hint about user object cached in the sessions

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Fixed tickets | fixes #issuenum
| Related PRs | sulu/sulu#prnum
| License | MIT

#### What's in this PR?

Add hint about user object cached in the sessions.

#### Why?

Did run into this issue after deploy a project.

### CODE REVIEW

File: cookbook/extend-entities.rst [hunk 1]
```diff
@@ -94,6 +94,11 @@ For the `User` entity (`se_users`):
                 model: App\Entity\User
                 repository: Sulu\Bundle\SecurityBundle\Entity\UserRepository
 
+.. note::
+
+   Symfony keeps the user object in the session, clearing the sessions is so sometimes required
```
Comment (wachterjohannes) [inline cookbook/extend-entities.rst:99]: add a link to the symfony command how to clear the sessions

**Review by wachterjohannes — CHANGES_REQUESTED** [review]

File: cookbook/extend-entities.rst (hunk 1 above)
Comment (alexander-schranz) [reply cookbook/extend-entities.rst:99]: @wachterjohannes dont know any command which clears the sessions.

Comment (wachterjohannes) [reply cookbook/extend-entities.rst:99]: how have you done it then in the project we had that problem?

Comment (alexander-schranz) [reply cookbook/extend-entities.rst:99]: Delete the session files manual they are stored differently based on the OS

**Pull request merged**